### PR TITLE
Improves all chapters

### PR DESCRIPTION
**Pull request by lftan**
Fixes grammars and rephrased some sentences for concisens.

### CODE REVIEW

File: src/intro.adoc [hunk 1]
```diff
@@ -40,24 +40,24 @@ microcontroller. An RPMI transport instance is associated to a particular
 RISC-V privilege level of the application processors and it must be accessed
 only by that RISC-V privilege level.
 
-*RPMI Messaging Protocol*: The RPMI messaging protocol includes different
-types of RPMI messages and the format each RPMI message type.
+*RPMI Messaging Protocol*: The RPMI messaging protocol defined different
```
Comment (pathakraul) [inline src/intro.adoc:43]: defines

Comment (lftan) [reply src/intro.adoc]: updated

File: src/srvgrp-device-power.adoc [hunk 2]
```diff
@@ -116,7 +116,7 @@ encoding for the power state.
 
 [#device-power-notifications]
 ==== Notifications
-This service group does not support any event for notification.
+This service group does not support event notifications.
```
Comment (pathakraul) [inline src/srvgrp-device-power.adoc:119]: This is giving the incorrect meaning

This section in all the service groups defines EVENTs which are available for notification or for notification message.  If NO event is defined then "no event for notification"

The changed line "does not support event notifications"  is talking more about the functionality - the Event Notifications functionality is not supported.

Comment (pathakraul) [reply src/srvgrp-device-power.adoc:119]: This comment applied to all the service groups

Comment (lftan) [reply src/srvgrp-device-power.adoc:119]: ok, will update this

File: src/srvgrp-system-suspend.adoc [hunk 3]
```diff
@@ -50,11 +50,11 @@ in the RISC-V SBI Specification cite:[SBI] for the `SUSPEND_TYPE` definition.
 
 [#system-suspend-notifications]
 ==== Notifications
-This service group does not support any event for notification.
+This service group does not support event notifications.
 
 ==== Service: SYSSUSP_ENABLE_NOTIFICATION (SERVICE_ID: 0x01)
-This service allows the application processor to subscribe to `SYSTEM_SUSPEND`
-service group defined events for notification. The platform can optionally
+This service allows the application processor to subscribe to events defined by
```
Comment (pathakraul) [inline src/srvgrp-system-suspend.adoc:56]: This format must then be used in all service groups *_ENABLE_NOTIFICATION service

Comment (pathakraul) [reply src/srvgrp-system-suspend.adoc:56]: I see only few service groups are changed. I also dont think the earlier line had issues.
Either one, Old or new, must be same for every service group

**Pull request merged**